### PR TITLE
@joeyAghion => Fix errant display of error message

### PR DESCRIPTION
**Pull request by mzikherman**
whoops!

### CODE REVIEW

**Comment by mzikherman** [comment]
This could definitely use tests...

File: lib/setup.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ export default function (app) {
     const isProduction = NODE_ENV === "production"
     const code = req.timedout ? 504 : (err.status || 500)
     const message = !isProduction ? err.message || err.text || err.toString() : ""
-    const detail = !isProduction ? err.stack && !isProduction : ""
+    const detail = !isProduction ? err.stack : ""
```
Comment (joeyAghion) [inline lib/setup.js]: Shouldn't this flip for clarity? `isProduction ? "" : err.stack`

Comment (mzikherman) [inline lib/setup.js]: Oh yea good call. I'll do that and try to move it into a middleware and a basic spec.

Comment (joeyAghion) [inline lib/setup.js]: OK but I'd accept just a flip if you want to leave the other improvements for another PR.
